`oscillator-overlap`: Make consistent with `oscillator` (#597)
precice/tutorials: changelog-entries/597.md

> - Offer Runge-Kutta 4 scheme and black-box time-stepping with RadauIIA scheme in oscillator-overlap case. Demonstrates use of time interpolation and dense output.

Reviewer: ```suggestion
- Added Runge-Kutta 4 scheme and black-box time-stepping with RadauIIA scheme as options to oscillator-overlap case. This demonstrates use of time interpolation and dense output.
```
